ops/gulp/dev-watch.js

```diff
@@ -3,23 +3,23 @@ const gulp = require('gulp');
 let environment = require('../lib/environment');
 
 const paths = {
-	src: 'src',
-	dest: !environment.production ? 'tmp' : 'dist'
+	src: 'scss',
+	dest: environment.production ? 'dist' : 'tmp'
 }
 
 const task = () => {
 
 	//watch docs templates and component templates
-	gulp.watch([`./docs/**/*.html`,`./src/**/*.html`]);
+	//gulp.watch([`./docs/**/*.html`,`./src/**/*.html`]);
 
 	//watch component styles
-	gulp.watch([`./src/**/*.scss`], ['pkg-css']);
+	gulp.watch([`./scss/**/*.scss`], ['pkg-css']);
 
 	//update docs styles
-	gulp.watch([`${paths.dest}/css/**/techne.css`, `${paths.dest}/css/**/techne-debug.css`], ['docs-css']);
+	gulp.watch([`${paths.dest}/techne.css`], ['docs-css']);
 
 	//update styleguide styles
-	gulp.watch([`./docs/styleguide/styles/**/*.scss`], ['docs-styleguide']);
+	//gulp.watch([`./docs/styleguide/styles/**/*.scss`], ['docs-styleguide']);
```

Comment:
@saadmhybris I commented this — `styleguide` no longer exists — and line 13 — nothing is triggered by the watch task.

The `docs-styleguide` task could be removed unless you'll need a custom SASS runner for the website later on.